src/components/sitepages/SiteData.vue

```diff
@@ -308,6 +388,7 @@ import CommandButton from '@/components/FormElements/CommandButton'
 import ImageInfoBar from '@/components/ImageDisplay/ImageInfoBar'
 import InfoImageThumb from '@/components/ImageDisplay/InfoImageThumb'
 import ThumbnailRow from '@/components/ImageDisplay/ThumbnailRow'
+import GroupImagesButton from '@/components/ImageDisplay/GroupImagesButton'
```

Comment:
Are we using the GroupImagesButton for this feature? I remember it being designed for the "exploding view" that we want to do later. Would it be safe to exclude from this PR?

Reply:
nope, deleted!